sdk/resourcemanager/azure-resourcemanager-monitor/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.27.0-beta.1 (2023-04-27)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Updated `api-version` to `2021-06-03-preview`.
```

Comment:
monitor's api-version is mixed, maybe the tag version `preview-2022-08`?

Reply:
Probably fine, not able to get too precise on these mixed ones.